<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetectionSensorForm</class>
 <widget class="QWidget" name="DetectionSensorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>170</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>170</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>240</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>240</width>
     <height>160</height>
    </size>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="labelSensorDescription">
       <property name="text">
        <string>io device desc</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelStatus">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="labelColorStatus">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
